vcpkg.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "nuklear",
+  "version-string": "0.0.1",
+  "port-version": 1,
+  "homepage": "https://github.com/Immediate-Mode-UI/Nuklear",
+  "description": "Minimal state immediate mode graphical user interface toolkit written in ANSI C and licensed under public domain.",
+  "dependencies": [
```

Comment:
Before going into the discussion about the general idea of getting rid of Python in favour of anything else (here cmake), I would like to point out that Nuklear has literally **zero dependencies**. It has only optional dependencies. So this is wrong. If vcpkg does not support optional dependencies, then we **must not** put any dependencies into the package description.

Reply:
Agreed. These dependencies should be defined in the demos rather than here :+1: